perfmetrics/scripts/ls_metrics/listing_benchmark.py

```diff
@@ -522,7 +531,7 @@ def _check_dependencies(packages) -> None:
     subprocess.call('bash', shell=True)
   log.info('Directory Structure Created.\n')
 
-  gcs_bucket = _mount_gcs_bucket(directory_structure.name)
+  gcs_bucket = _mount_gcs_bucket(directory_structure.name, args.gcsfuse_flags[0])
```

Comment:
Can we validate the length of gcsfuse_flags before access the 0th index?

Reply:
gcsfuse_flags is a required flag and has to be passed either empty or with some value